### PR TITLE
Add cassandra framework

### PR DESCRIPTION
**Pull request by BenWhitehead**

### CODE REVIEW

**Comment by BenWhitehead** [comment]
Still need to test installing this will try tomorrow.

**Comment by BenWhitehead** [comment]
Tested by installing and running cli in a docker container

```
docker run -i -t --net=host -v `pwd`:/tmp/universe ubuntu /bin/bash
```

once the docker container has started the following commands were run (in order)
```bash
mkdir /tmp/dcos-cli
cd tmp/
apt-get update && apt-get install -y wget curl python python-pip git
pip install virtualenv
wget https://raw.githubusercontent.com/mesosphere/install-scripts/master/dcos-cli/install.sh
chmod +x install.sh
./install.sh $SECRET_KEY /tmp/dcos-cli localhost 8080
source /tmp/dcos-cli/bin/env-setup

dcos config set marathon.host localhost
dcos config set marathon.port 8080
dcos config set package.cache /usr/local/dcos/registry
dcos config set package.sources ['file:///tmp/universe', 'git://github.com/mesosphere/universe.git']

echo '{"mesos/master":"zk://localhost:2181/mesos","cassandra/zk":"zk://localhost:2181/cassandra-mesos/stage","framework/name":"stage"}' > /tmp/cassandra-config.json

dcos package install cassandra --options=/tmp/cassandra-config.json
```

File: repo/packages/C/cassandra/0/marathon.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "cpus": 0.5,
+  "mem": 512,
+  "ports": [
+    0
+  ],
+  "uris": [
+    "https://downloads.mesosphere.io/cassandra-mesos/artifacts/0.1.0-SNAPSHOT-390-rewrite-825756a084/cassandra-mesos-0.1.0-SNAPSHOT-390-rewrite-825756a084.tar.gz",
+    "https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz"
+  ],
+  "labels": {
```
Comment (ConnorDoyle) [inline repo/packages/C/cassandra/0/marathon.json]: Please remove all of these labels.  When using the DCOS CLI, these and more will be added automatically.  Other clients probably don't want them either.

File: repo/packages/C/cassandra/0/package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "cassandra",
+  "version": "0.1.0-SNAPSHOT",
+  "scm": "https://github.com/mesosphere/cassandra-mesos.git",
+  "maintainer": "support@mesosphere.io",
+  "description": "Apache Cassandra running on Apache Mesos",
+  "framework": true,
+  "tags": ["mesosphere", "framework", "cassandra"]
```
Comment (ConnorDoyle) [inline repo/packages/C/cassandra/0/package.json]: Probably no need to duplicate the name "cassandra" as a tag here.

**Comment by ConnorDoyle** [comment]
Squash when you rebase please :pray:

**Comment by BenWhitehead** [comment]
Squashed and rebased!

**Comment by ConnorDoyle** [comment]
:shipit:

**Pull request merged**